Automattic/wp-calypso
# Only show the option to make primary for staging domains

#### Changes proposed in this Pull Request

We don't want to allow any management options of staging domains other than to make them primary.

Currently when you go to the management page for a staging domain, you have the options to set email and edit DNS Records. Neither of these options are valid for staging domains. We only want to allow for the option to make it primary.

Currently you will see this:

<img width="753" alt="Screen Shot 2019-11-21 at 8 45 02 AM" src="https://user-images.githubusercontent.com/1379730/69343744-29f95a80-0c3c-11ea-8b4b-6a454a44b43f.png">

After this patch, you will see:

<img width="753" alt="Screen Shot 2019-11-21 at 9 24 08 AM" src="https://user-images.githubusercontent.com/1379730/69346480-d6d5d680-0c40-11ea-8a51-5d385dca2105.png">

#### Testing instructions

Depends on D35724-code

- Make sure that you don't have the option to manage anything except for making a staging domain primary.

- Make sure that you still have the option to rename a .blog or wpcom subdomain.
